config/environments/production.rb

```diff
@@ -67,6 +67,8 @@
   host = ENV["HEROKU_APP_NAME"].present? ? "#{ENV['HEROKU_APP_NAME']}.herokuapp.com" : Rails.application.credentials.domain
   config.action_mailer.default_url_options = { host: host }
 
+  ENV["VUE_APP_API_BASE_URL"] = "https://#{host}"
```

Comment:
Nice and easy. This happens before the dockerfile runs then?

Reply:
@jhash so when I added this commit https://github.com/WorkHands/RAPiDSkills/pull/131/commits/87c79ebda4940f9004c35d86cdab80f03d883cb5, I end up with URLS like: http://rapid-skills-favorites-wxsaqyz/undefined/standards. So it is clearly reading the base URL environment variable correctly, but I can't figure out where that undefined is coming from. When it was reading the base URL from the `vue/.env` file then there was no undefined.

Reply:
@littleforest this is the only other thing in that URL but it's defined right before: ```export const API_NAMESPACE = '/api/v1';
const apiUrl = `${VUE_APP_API_BASE_URL}${API_NAMESPACE}`;```